src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return carCost * days;
+  } else if (days >= 3 && days < 7) {
+    return (carCost * days) - smallDiscount;
+  }
+
+  return (carCost * days) - bigDiscount;
```

Comment:
that looks strange that bigDiscount is default.
Let's start with checking for a big discount - and as a default we will just return totalPrice